packages/jest-circus/src/run.ts

```diff
@@ -126,12 +117,8 @@ const _runTest = async (test: Circus.TestEntry): Promise<void> => {
     await _callCircusTest(test, testContext);
   }
 
-  if (test.parent.errors.length === 0) {
```

Comment:
This check is no longer neccesary if `_runTest` is being skipped if there are setup errors. All `afterEach` hooks should still run if there are any test errors when `_runTest` is called.